### PR TITLE
LocationService Fix for Android

### PR DESCRIPTION
**Pull request by ah1053**
1. Android wait for out of focus and then check for permission.
2. Add comments for public functions.
3. Fix indentation.

### CODE REVIEW

File: Runtime/Scripts/LocationService.cs [hunk 1]
```diff
@@ -23,194 +23,223 @@
...
-                Altitude = info.altitude,
-                Course = 0,
-                Speed = 0,
-                Timestamp = ts
-            };
-            return loc;
+          Logger.Log("Initializing Location Service Timed Out");
+          Input.location.Stop();
+          initLocationServiceException = new LocationException("Initializing Location Service Timed Out");
+          yield break;//Exception will be thrown from RetrieveLocation()
```
Comment (lgarner) [inline Runtime/Scripts/LocationService.cs:152]: Space.

**Review by lgarner — APPROVED** [review]
ok.

**Review by franklin-huang-mobiledgex — APPROVED** [review]
cool thanks! There are other files with this spacing issue as well, right?

**Comment by ah1053** [comment]
Yeah, will do some PR later for fixing indentation for other files.
Will track it here https://mobiledgex.atlassian.net/browse/EDGECLOUD-5292

**Pull request merged**